Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM golang:1.19 AS build
+
+ARG taskfileVersion="3.20.0"
+ARG taskfileChecksum="75cd08890ff18f6036255c7630aa17f9ea81bd6b6166747a3913bbb1cff1357c"
+
+WORKDIR /tmp
+RUN wget https://github.com/go-task/task/releases/download/v${taskfileVersion}/task_linux_amd64.deb \
+	&& echo "${taskfileChecksum}  task_linux_amd64.deb" > checksums.txt \
+	&& sha256sum --check checksums.txt \
+	&& dpkg -i task_linux_amd64.deb \
+	&& rm -f task_linux_amd64.deb
+
+WORKDIR /work
+COPY go.mod go.sum ./
+RUN go mod download && go mod verify
+
+COPY . ./
+RUN task build
+
+
+FROM debian:stable-slim
```

Comment:
Was there some specific reason why we need this image specifically?

Reply:
"this image" meaning debian? Could go alpine but then we'd also need to build the binary in a musl libc container like alpine, and the official golang images are debian-based.